docs/user/artifacts/function_list

```diff
@@ -1066,6 +1075,26 @@ listMultiply(factor, list): listMultiply
 
 * listMultiply: String 1:1 required
 
+litem(source, index): result
+''''''''''''''''''''''''''''
+
+* source: List 1:1 required
+
+* index: Integer 1:1 required
+
+* result
+
+literate(index, times, expression): result
+''''''''''''''''''''''''''''''''''''''''''
```

Comment:
The description from the NAME should show up here? `List of items resulted from iteration.`

We may need to rerun the generator from https://github.com/geotools/geotools/pull/2216